include/aos/common/tools/log.hpp

```diff
@@ -189,9 +298,9 @@ class Log : private NonCopyable {
     }
 
     StaticString<cMaxLineLen> mLogLine;
+    StaticString<cMaxLineLen> mFieldsLine;
```

Comment:
Let's have dedicated constant for this: cMaxFieldsLen (AOS_CONFIG_LOG_FIELDS_LEN)

Reply:
Done